### ISSUE TITLE
Compute shader group just ignore rest of shaders if a suffix doesnt appear

### ISSUE THREAD

**Issue by GeForceLegend** [IrisShaders/Iris#1840]
### What happened?

In a group of compute shaders (for example composite.csh to composite_z.csh), if a shader with certain suffix does not exist, the rest shaders of this group is just ignored (if _e is missing, _f to _z all will not load).
While testing shaders, just like developers may delete some screen shaders for some reason, its common that part of compute shaders in a group get deleted. This is annoying while testing some effects using compute shaders.

### Minecraft Version

Minecraft 1.19.3

### Iris Version

iris-mc1.19.3-1.5.0.jar

### Operating System

Windows 10

### What is your GPU?

NVIDIA Quadro P5200

**Comment by IMS212** [comment]
Yes, this is done to improve load times. Might revert this if this actually poses a problem.

**Comment by GeForceLegend** [comment]
> Yes, this is done to improve load times. Might revert this if this actually poses a problem.

If you really want to improve load time, scanning current files in target path instead of finding if there are target files will be much faster, and should not affect performance too much when expanding target list

**Comment by GeForceLegend** [comment]
Don't know if you have looked into my LSP, but I think my way is better than searching if shader files exist in a hashset:
Reading all file names in expected folder (like `shaders/` and `shaders/worldX`), using someway like regex to scan if it's a valid shader file (can check it's type and index at the same time), this can limit scans required to file count but not theoretical possible shader count, reducing scan time at most cases, without ignoreing any valid shader.
